ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```diff
@@ -36,23 +36,24 @@ else
 fi
 
 pip install .
-export  EXTRA_FLAGS=""
+export EXTRA_FLAGS=" --lookback ${LOOKBACK}d"
```

Comment:
If we're changing this anyway, can we remove the days here change it to be `--lookback ${LOOKBACK}` with `LOOKBACK=15d` ?

Reply:
That will restrict it to a hardcoded value of 15d. The idea was to have it as default, at the same time provide others with flexibility to override it.

Reply:
No, I'm not saying to hardcode 15d, I am saying to not hardcode "d".  Anyway, I don't think it's a big deal bc would we ever filter out runs on number of hours? Probably never